### PR TITLE
Criteo (Certified Connector)

### PR DESCRIPTION
**Pull request by gabrigiunchi**
image

---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [X] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [X] I attest that the connector works and I verified by deploying and testing all the operations. 
- [X] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [X] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [X] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [X] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

File: certified-connectors/Criteo/api.properties [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "connectionParameters": {
+    "token": {
+      "type": "oauthSetting",
+      "oAuthSettings": {
+        "identityProvider": "oauth2",
+        "clientId": "criteo_client_id",
```
Comment (gabrigiunchi) [inline certified-connectors/Criteo/api.properties:7]: not sure if I have to also put "clientSecret": ""

**Comment by Amjed-Ayoub** [comment]
Hello @gabrigiunchi,

Thank you so much for submitting your connector artifacts to our open source repository.
Can you please confirm if you own the API for this connector?

I look forward to hearing form you.

Thank you so much again.

File: certified-connectors/Criteo/api.properties (hunk 1 above)
Comment (Amjed-Ayoub) [reply certified-connectors/Criteo/api.properties:7]: @gabrigiunchi No need to add the clientSecret :-)

**Comment by gabrigiunchi** [comment]
> Hello @gabrigiunchi,
> 
> Thank you so much for submitting your connector artifacts to our open source repository. Can you please confirm if you own the API for this connector?
> 
> I look forward to hearing form you.
> 
> Thank you so much again.

Hello Amjed,

No, I don't own the Criteo API

**Comment by Amjed-Ayoub** [comment]
> > Hello @gabrigiunchi,
> > Thank you so much for submitting your connector artifacts to our open source repository. Can you please confirm if you own the API for this connector?
> > I look forward to hearing form you.
> > Thank you so much again.
> 
> Hello Amjed,
> 
> No, I don't own the Criteo API

Hello @gabrigiunchi,

Thank you so much for confirming this with me.

You still can publish this connector as an Independent Publisher. You will need to move your connector folder to our "independent-publisher-connectors" folder in this repository.
Please follow the certification process below.
1.	Please read and agree to our [manifesto](https://github.com/microsoft/PowerPlatformConnectors/wiki/Independent-Publisher-Connector-Group-%22Manifesto%22).
2.	Ensure that your connector complies with Microsoft requirements, as specified in [Prepare the connector artifacts.](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission-ip#prepare-and-submit-your-connector-for-certification)
3.	Ensure that your connector [contains valid metadata](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#step-3-add-metadata).
4.	Please open source your connector files in our [GitHub repository](https://github.com/microsoft/PowerPlatformConnectors/tree/dev/independent-publisher-connectors). Our Certification Engineers will review within 1-2 weeks, and you will be notified in GitHub. Please include the below files in your pull request.
•	apiDefinition.swagger.json
•	apiProperties.json
•	Readme.md (you can find an example [here](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/readme.md) and for raw view for code [here](https://raw.githubusercontent.com/microsoft/PowerPlatformConnectors/dev/templates/Independent%20Publisher/readme.md))
•	Screenshots of the connector testing.
5.	Microsoft will test your connector, and we expect all tests to be completed within 1-2 weeks.
6.	If your connector passes all tests, the deployment process begins, and it typically takes up to 3-4 weeks to deploy your connector to all our regions.
The overall process will take up to 5-8 weeks to complete. For detailed steps and instructions, please see our [Certification process](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission-ip) page.

I hope that helps.

Please let me know if you have any questions.

Thank you so much again.

Best regards,
Amjed ayoub

**Pull request closed without merging**